### PR TITLE
Style Snapshot Component (Head) 

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR integrates the core styling requried for the Snapshot Head Component.

Outcome: 

image

_Note the failure group component is yet to be developed, so "Group" is our placeholder._

Excepted Result:

image

### CODE REVIEW

**Review by holounic — APPROVED** [review]
LGTM!

btw it looks very cool

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+:root {
+  --failed: #DC4F4F;
```
Comment (krasimir-google) [inline src/main/webapp/style.css:2]: please use lowercase for the constant: https://google.github.io/styleguide/htmlcssguide.html#Capitalization

**Review by krasimir-google — APPROVED** [review]
LGTM with a couple of nits

File: src/main/webapp/style.css (hunk 1 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css]: Addressed here d029e00

**Pull request merged**